Zend/zend_vm_execute.h

```diff
@@ -5935,7 +5936,13 @@ static ZEND_OPCODE_HANDLER_RET ZEND_FASTCALL ZEND_FETCH_CONSTANT_SPEC_CONST_CONS
 				}
 			}
 
-			if (EXPECTED((value = zend_hash_find(&ce->constants_table, Z_STR_P(EX_CONSTANT(opline->op2)))) != NULL)) {
```

Comment:
You're directly modifying zend_vm_execute here. You should change zend_vm_def instead and regenerate using zend_vm_gen.php